frontend/src/routing/PrivateRoute.js

```diff
@@ -9,23 +9,24 @@ const PrivateRoute = ({
   ...rest
 }) => (
   <Route
+    // eslint-disable-next-line react/jsx-props-no-spreading
     {...rest}
-    render={(props) =>
-      !isAuthenticated && !loading ? (
-        <Redirect to='/login' />
-      ) : (
-        <Component {...props} />
-      )
-    }
+    render={props => (!isAuthenticated && !loading ? (
+      <Redirect to="/login" />
```

Comment:
Avoid negative conditioning. Also, keep these route in the config for better maintainability